src/library/Uno.Material/Styles/Controls/RadioButton.xaml

```diff
@@ -58,6 +64,39 @@
 							<VisualStateGroup x:Name="CheckStates">
 
 								<VisualState x:Name="Checked">
+									<Storyboard>
+										<DoubleAnimation Storyboard.TargetName="PressRingTransform"
+														 Storyboard.TargetProperty="ScaleX"
+														 Duration="0:0:0.4"
+														 EasingFunction="{StaticResource EaseInOutFunction}"
+														 From="0"
+														 To="1" />
+
+										<DoubleAnimation Storyboard.TargetName="PressRingTransform"
+														 Storyboard.TargetProperty="ScaleY"
+														 Duration="0:0:0.4"
+														 EasingFunction="{StaticResource EaseInOutFunction}"
+														 From="0"
+														 To="1" />
+
+										<android:DoubleAnimation Storyboard.TargetName="PressRing"
```

Comment:
why android specific?

Reply:
Because Uno doesn't support DoubleAnimationUsingKeyFrame yet.

Reply:
Let's add a comment and link the issue then :) And what will happen when it does support it? Seems like suddenly an error will occur because two animations will try trigger on PressRing for Opacity.